# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/filter-effects/svg-multiple-filter-functions.html.ini
# Commit: beec388d4b0e
# Full Hash: beec388d4b0e6a50ac243ed86c0e3f1f48cbe744
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-06-25 09:36:02
# Regressor Bug: 1896503
# File Overlap Count: 1
# Description:
#   Bug 1896503 - Implement FilterInstance code to send SVG filter graph to WebRender r=mstange,gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D174208
# ==============================================================================

diff -r 2147e0051e23 -r beec388d4b0e testing/web-platform/meta/css/filter-effects/svg-multiple-filter-functions.html.ini
--- a/testing/web-platform/meta/css/filter-effects/svg-multiple-filter-functions.html.ini	Mon Jun 24 23:40:38 2024 +0000
+++ b/testing/web-platform/meta/css/filter-effects/svg-multiple-filter-functions.html.ini	Mon Jun 24 23:45:05 2024 +0000
@@ -1,2 +1,4 @@
 [svg-multiple-filter-functions.html]
-  expected: FAIL
+  fuzzy:
+    maxDifference=0-1;totalPixels=0-10000
+
